File: lessons/lesson01/codes/nq005_salary.py
# NQ005: 工资
# 读取员工编号、工作时数、时薪，计算工资

number = int(input())
hours = int(input())
rate = float(input())
salary = hours * rate
print(f"NUMBER = {number}")
print(f"SALARY = U$ {salary:.2f}")

File: lessons/lesson01/codes/nq005_salary.cpp
// NQ005: 工资
#include <cstdio>

int main() {
    int number, hour;
    double money;
    scanf("%d%d%lf", &number, &hour, &money);
    printf("NUMBER = %d\n", number);
    printf("SALARY = U$ %.2lf\n", hour * money);
    return 0;
}
